IBPSA/Fluid/Sources/BaseClasses/PartialSource_volume.mo

```diff
@@ -0,0 +1,65 @@
+within IBPSA.Fluid.Sources.BaseClasses;
+partial model PartialSource_volume
+  "Partial component source with one fluid connector"
```

Comment:
It would be good if the comment specifies how this component differs from the regular `PartialSource`